configs/eth/withdrawal_mainnet.properties

```diff
@@ -5,8 +5,10 @@ withdrawal.notaryIrohaAccount=notary@notary
 withdrawal.notaryListStorageAccount=notaries@notary
 withdrawal.notaryListSetterAccount=notary@notary
 withdrawal.registrationIrohaAccount=eth_registration_service@notary
-withdrawal.tokenStorageAccount=eth_token_storage_service@notary
-withdrawal.tokenSetterAccount=eth_token_storage_service@notary
+withdrawal.ethAnchoredTokenStorageAccount=eth_anchored_token_service@notary
+withdrawal.ethAnchoredTokenSetterAccount=eth_token_service@notary
+withdrawal.irohaAnchoredTokenStorageAccount=iroha_anchored_token_service@notary
+withdrawal.irohaAnchoredTokenSetterAccount=eth_token_service@notary
```

Comment:
I see a lot of properties are the same in different environments. Maybe create a separate configuration file? It will be the same for every environment and it will be easier to maintain.

Reply:
It is better to have a one config file for one service. I think the best option is to have a one "default" property file and remove environment-depended files. But it is out of scope.